server/sv-init.lua

```diff
@@ -0,0 +1,34 @@
+STORIES_CACHE = {
+    NEWS = {},
+    SENTENCES = {},
+}
+
+CreateThread(function()
+    if IsResourceOnServer('rcore_prison') then
+        print('^3[API] ^0rcore_prison detected, enabling jail stories.')
+
+        AddEventHandler('rcore_prison:prisonerState', function(data)
+            local state = data and data.state
+            local playerId = data and data.serverId
+            local prisonerData = data and data.prisoner
+
+            if state == 'jailed' and prisonerData then
+                local jailTime = prisonerData and prisonerData.jailTime or 0
+                local prisonerName = prisonerData and prisonerData.name
+
+                -- TODO: Requires to modify frontend with some variable with prefix jail format (seconds/mins/hours/days) to make proper sense.
```

Comment:
Something you need from me? `ps-core` is going to come with utility functions for formatting dates, so we can use that if that was the intention?

Reply:
The thing is now that is strictly to months in frontend when you see, related to create definition for locales of UI? 🤔